fedimintd/src/bin/distributedgen.rs

```diff
@@ -88,29 +95,37 @@ async fn main() {
             address,
             base_port,
             name,
+            password,
         } => {
-            println!("{}", gen_tls(&dir_out_path, address, base_port, name));
+            let salt: [u8; 16] = rand::random();
+            fs::write(dir_out_path.join(SALT_FILE), hex::encode(salt)).expect("write error");
+            let key = get_key(password, dir_out_path.join(SALT_FILE));
```

Comment:
~Eventually we can probably devise a more compact file format (one binary encoded file instead of two), but for now this seems good.~

Ah, that's a different salt for the password. Actually more of a pepper since it's unique to the user and kinda private.